index.php

```diff
@@ -108,6 +108,13 @@
 	 */
 	$hyphaContentLanguage = $O_O->getContentLanguage();
 
+		/*
+		*	load a list of articles that need a review
+		*/
+		$hyphaHtml->writeToElement('review', hypha_getReview($hyphaContentLanguage));
```

Comment:
This is a nice feature, although the logic for `a list of articles that need a review` should be in the `peer_reviewed_article`.